notebooks/community/pipelines/google_cloud_pipeline_components_bqml_pipeline_demand_forecasting.ipynb

```diff
@@ -706,6 +706,7 @@
         "KFP_COMPONENTS_PATH = \"components\"\n",
```

Comment:
<p>Looks like format is an older template.</p><p>Break the Overview ... Costs into 4 separate text cells.</p><p>Move Dataset to after objective.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1439/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>done</p>